src/gui/wizard/owncloudadvancedsetuppage.cpp

```diff
@@ -65,13 +67,13 @@ OwncloudAdvancedSetupPage::OwncloudAdvancedSetupPage()
     });
     connect(_ui.bSelectiveSync, &QAbstractButton::clicked, this, &OwncloudAdvancedSetupPage::slotSelectiveSyncClicked);
 
+    const auto theme = Theme::instance();
     QIcon appIcon = theme->applicationIcon();
-    _ui.lServerIcon->setText(QString());
-    _ui.lServerIcon->setPixmap(appIcon.pixmap(48));
-    _ui.lLocalIcon->setText(QString());
-    
-    // TO DO: File doesn't exist anymore - unneccessary or replacement needed?
-    _ui.lLocalIcon->setPixmap(QPixmap(Theme::hidpiFileName(":/client/theme/folder-sync.png")));
+    int appIconSize = 64;
+    if (Theme::isHidpi()) {
+        appIconSize *= 2;
```

Comment:
I'd const auto appIconSize and use the ternary operator:
`const auto appIconSize = Theme::isHidpi() ? 128 : 64;`
or if *really* you want to keep the factor two:
`const auto appIconSize = 64 * (Theme::isHidpi() ? 2 : 1);`

Reply:
This way it's a lot cleaner:) thanks